pywebdriver/plugins/escpos_driver.py

```diff
@@ -218,6 +218,12 @@ def __init__(self, *args, **kwargs):
         def get_vendor_product(self):
             return "escpos-icon"
 
+        def printer_available(self):
```

Comment:
I think either the function should be named something like `is_usb_printer_available` or take into account the device_type

Reply:
I added the check for the device_type. The default False should result in the same behaviour as currently if not usb. I can't really test it currently for the other types.

Do you know for serial and win32 a good check to do it?

Reply:
> Do you know for serial and win32 a good check to do it?

I would need to have an environment to test it, but I don't have a windows computer or VM ready right now. I think we may need to wait for this before merging

For serial I'm not sure if anyone uses it tbh.